sdcm/mgmt.py

```diff
@@ -243,6 +243,15 @@ def detailed_progress(self):
         relevant_key = [key for key in parsed_progress_table.keys() if parsed_progress_table[key]][0]
         return parsed_progress_table[relevant_key]
 
+    def wait_for_task_success_status(self, timeout=10800):
+        start_time = time.time()
+        while start_time + timeout > time.time():
```

Comment:
it may be good to add some sleep in this loop. even of 4 seconds in order not to run unneeded extra commands and explode the logs.